src/lib/chat/matrix-client.ts

```diff
@@ -408,7 +414,9 @@ export class MatrixClient implements IChatClient {
     };
   }
 
-  private mapChannel = (room: Room): Partial<Channel> => this.mapToGeneralChannel(room);
+  private mapChannel = (room: Room): Partial<Channel> => {
```

Comment:
How come the return and brackets were added here? Doesn't it work the same ?

Reply:
ah, good catch. I was trying something here, but i had removed it. forgot to remove the brackets. Will update in my latest PR.